influxdb3_write/src/write_buffer/buffer_segment.rs

```diff
@@ -86,6 +86,11 @@ impl OpenBufferSegment {
         self.segment_writer.write_batch(write_batch)
     }
 
+    #[allow(dead_code)]
```

Comment:
I think this could be `#[cfg(test)]` to be more specific. I am not sure if you can apply the compiler flag to the function directly, or if that requires it go in its own separate impl block though.

Reply:
You should be able to apply `#[cfg(test)]` here just fine. It's just a cfg directive 😄